nautobot/core/templates/generic/object_update.html

```diff
@@ -0,0 +1,2 @@
+{% extends 'generic/object_create.html' %}
+{% comment %}You should re-write your own model '{object}_update.html' template if you are not using the same template for creating an object and updating an object. {% endcomment %}
```

Comment:
Right now the logic falls back to `generic/object_update.html`. But it would make sense to fall back to `<object>_create.html` I will make sure to include the logic